engine/graphics/src/opengl/graphics_opengl.cpp

```diff
@@ -159,6 +159,7 @@
     #if defined GL_ES_VERSION_2_0
         #undef GL_ARRAY_BUFFER_ARB
         #undef GL_ELEMENT_ARRAY_BUFFER_ARB
+        #define GL_HALF_FLOAT_OES 0x8D61
```

Comment:
In fact, for WebGL we should use header files from ES 2.0/3.0, because it is based on it. But someone decided to use files from desktop GL. That's strange. And there is no GL_HALF_FLOAT_OES in desktop GL. So I added this constant manually as #define.

PS If the desktop include above is replaced by include gl2ext.h, for example, the webgl renderer stops working and pulls a huge amount of patching and testing.

Reply:
I think you should put this into graphics_opengl_defines.h and follow the style of the other defines

Reply:
Okay 🙌🏻 Done.